lib/elixir/lib/calendar/datetime.ex

```diff
@@ -998,6 +1004,73 @@ defmodule DateTime do
     naive_diff + System.convert_time_unit(offset_diff, :second, unit)
   end
 
+  @doc """
+  Adds a specified amount of time to a `DateTime`.
+
+  Accepts an `integer` in any `unit` available from `t:System.time_unit/0`.
```

Comment:
Nitpick: I think `` `integer` `` here was referring to the second argument, which is now `amount_to_add`. We could either remove the backticks or rephrase this paragraph :)